apps/03TicTacToe.js

```diff
@@ -3,55 +3,86 @@
 var assert = require('assert');
 var readline = require('readline');
 var rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
+    input: process.stdin,
+    output: process.stdout
 });
+
 var board = [
-  [' ', ' ', ' '],
-  [' ', ' ', ' '],
-  [' ', ' ', ' ']
+    [' ', ' ', ' '],
+    [' ', ' ', ' '],
+    [' ', ' ', ' ']
 ];
 
 var playerTurn = 'X';
 
+function toggleTurn() {
+    if (playerTurn === 'X') {
+        playerTurn = 'O';
+    } else {
+        playerTurn = 'X';
+    }
```

Comment:
For the sake of the lesson, toggle the value of playerTurn using a ternary operator.